intranet/apps/seniors/management/commands/cleanup_destinations.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand
+from ...models import Senior
+
+
+class Command(BaseCommand):
+    help = "Clean up senior destinations"
+
+    def handle(self, *args, **options):
+        if len(Senior.objects.all()) > 0:
```

Comment:
Replace with `if Senior.objects.count() > 0:` to avoid loading all the objects into memory.